### PR TITLE
Update test config

### PR DESCRIPTION
**Pull request by ulgens**
## Description

Update Python and Django versions in test workflow.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [x] Enhancement/optimization
- [ ] Documentation

## Issues Fixed or Closed by This PR

* - 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the CONTRIBUTING document.
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by botberry** [comment]
Hi, thanks for contributing to Strawberry 🍓!

We noticed that this PR is missing a `RELEASE.md` file. We use that to automatically do releases here on GitHub and, most importantly, to PyPI!

So as soon as this PR is merged, a release will be made 🚀.

Here's an example of `RELEASE.md`:

```markdown
Release type: patch

Description of the changes, ideally with some examples, if adding a new feature.
```

Release type can be one of patch, minor or major. We use [semver](https://semver.org/),so make sure to pick the appropriate type. If in doubt feel free to ask :)

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -10,7 +10,7 @@ jobs:
     strategy:
       matrix:
         python-version: ['3.7', '3.8', '3.9']
-        django: [3.1.3, 3.0.2, 2.2]
+        django: [3.1.4, 3.0.11, 2.2]
```
Comment (ulgens) [inline .github/workflows/test.yml:13]: I think we should add patch version to 2.2 too.

Comment (patrick91) [reply .github/workflows/test.yml:13]: let's do that :)

Comment (ulgens) [reply .github/workflows/test.yml]: Done.

**Comment by ulgens** [comment]
@patrick91 I think repo settings should be after too.

**Comment by patrick91** [comment]
> @patrick91 I think repo settings should be after too.

what do you mean?

**Comment by ulgens** [comment]
@patrick91 Github says it waits for "Test on 3.8 - django 3.0.2" test for example, as required check. That test combination doesn't exist anymore.

**Review by patrick91 — APPROVED** [review]

**Pull request merged**